sagemaker-fundamentals/create-training-job/create_training_job_vpc.ipynb

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
<p>Not sure why, but my output was duplicates. I checked all the ServiceNames and for some reason, s3 is the only duplicate.</p><pre class="ql-syntax" spellcheck="false">com.amazonaws.us-east-1.s3
com.amazonaws.us-east-1.s3
</pre><p>You could deduplicate before printing to avoid customer confusion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>I noticed that too. I had <code>com.amazonaws.us-west-2.s3</code>  twice. </p>